src/Table/Grid.php

```diff
@@ -411,7 +411,7 @@ protected function buildColumn($name, $callback = null)
      */
     protected function buildModel($model)
     {
-        if ($this->paginate === true && method_exists($model, 'paginate')) {
+        if ($this->isEloquentModel($model) && $this->paginate) {
```

Comment:
Ah I see, ok I'll fix that! What about using `is_callable()` like in my recent commit instead of using `method_exists()`?